### DISCUSSION TITLE
在使用双拼的情况下 关闭全拼输入方式

### DISCUSSION THREAD

**Discussion by awx-dylan-xie** [rime/home#1105] [Q&A]
想问如何在使用双拼的情况下避免出现以下的情况（触发了全拼的输入）
<img width="762" alt="image" src="https://user-images.githubusercontent.com/87410542/181487298-0f850d5f-c88a-463c-9fb2-8c49d091c8e7.png">

**Comment by LEOYoon-Tsaw** [comment]
問你的方案製作者

**Reply by awx-dylan-xie** [reply]
用的是官方的小鹤双拼，虽然有做一些调整

**Reply by awx-dylan-xie** [reply]
检查了一下 应该是custom里面engine/translators下面多了table_translator引发的问题 注释掉之后好像没有引起其他的变化 但解决了这个问题（虽然这个custom也基本上是从别处复制来的 并不太理解这一行修改了什么
<img width="773" alt="image" src="https://user-images.githubusercontent.com/87410542/181611091-fedc39c1-2b53-47e5-bb09-0d9258e9593d.png">
